### PR TITLE
(DOCSP-27230): Update SwiftUI tutorial for v2 template app

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

- https://jira.mongodb.org/browse/DOCSP-27230

### Staged Changes

- [SwiftUI iOS Tutorial](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-27230/tutorial/swiftui/)

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-realm update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/tutorial/swiftui.txt [hunk 1]
```diff
@@ -21,12 +21,20 @@ the creation of a to-do list application. This application enables users to:
...
 - Register their email as a new user account.
 - Sign in to their account with their email and password (and sign out later).
-- View, create, modify, and delete tasks.
+- View, create, modify, and delete their own tasks.
+- View all tasks, even where the user is not the owner.
 
-This tutorial adds on to the Template App. You will add a new ``priority`` field 
+The template app also provides a toggle that simulates the device being in 
+"Offline Mode." This toggle lets you quickly test Device Sync functionality 
+on a simulator, simulating the user having no internet connection. However,
```
Comment (mongodben) [inline source/tutorial/swiftui.txt:29]:
```suggestion
on a simulator, simulating the user having no internet connection. However,
```

"simulator, simulating" reads odd to me. perhaps 'simulating' could be replaced by a synonym like 'mocking', 'emulating', etc.

File: source/tutorial/swiftui.txt [hunk 2]
```diff
@@ -21,12 +21,20 @@ the creation of a to-do list application. This application enables users to:
...
+you would likely remove this toggle in a production application.
+
+This tutorial builds on the Template App. You will add a new ``priority`` field 
 to the existing ``Item`` model and update the 
 :ref:`Flexible Sync subscription <queryable-fields>` to only show items within 
-a range of priorities. 
+a range of priorities. This is an arbitrary example to illustrate how you might 
+adapt the template app for your own needs. You would not realistically make 
+this specific change given the current structure of the template app.
```
Comment (mongodben) [inline source/tutorial/swiftui.txt:37]: like i agree w you here, but i think we could *soften* the language to smthn like:

```suggestion
a range of priorities. This example illustrates how you might 
adapt the template app for your own needs. You would not necessarily make 
this change given the current structure of the template app. 
```

File: source/tutorial/swiftui.txt [hunk 3]
```diff
@@ -105,7 +114,8 @@ Explore the Template App
            - This file uses the values from ``AppConfig.swift`` to initialize
              the ``realmApp``. The ``realmApp`` is how your app communicates 
              with the App Services backend. This provides access to login and 
-             authentication.
+             authentication. This file also contains the error handler that 
+             listens for Sync errors.
```
Comment (mongodben) [inline source/tutorial/swiftui.txt:118]:
```suggestion
             listens for Device Sync errors.
```

File: source/tutorial/swiftui.txt [hunk 4]
```diff
@@ -351,31 +362,34 @@ Change the Subscription
       Return to your mobile app. Because we added ``reRunOnOpen``, the app
       should re-sync only the documents that match the Flexible Sync query.
       After an initial moment when Realm resyncs the document collection, you 
-      will see the new Item of High priority that you created.
+      will see only the new Item of High priority that you created.
```
Comment (mongodben) [inline source/tutorial/swiftui.txt:365]:
```suggestion
      will only see the new Item of High priority that you created.
```

nit: not sure why but this reads a lil better to me

File: source/tutorial/swiftui.txt [hunk 5]
```diff
@@ -351,31 +362,34 @@ Change the Subscription
...
+
+      You'll note that the document you initially created is not synced,
```
Comment (mongodben) [inline source/tutorial/swiftui.txt:367]:
```suggestion
      The Item document you initially created is not synced, 
```

File: source/tutorial/swiftui.txt [hunk 6]
```diff
@@ -351,31 +362,34 @@ Change the Subscription
...
 
-      Realm creates the Item locally, syncs it with the backend, and then reverts 
-      the write because it doesn't meet the subscription rules.
+      You can also see this message in the console log.
 
-      You'll note, too, that the document you initially created is not synced, 
-      because it does not have a ``priority`` field. If you want this Item to 
-      be synced, you can edit the document in the Atlas UI and add a value for 
-      the priority field.
+      In this instance, Realm creates the Item locally, syncs it with the
```
Comment (mongodben) [inline source/tutorial/swiftui.txt:386]:
```suggestion
      In this situation, Realm creates the Item locally, syncs it with the 
```

nit: had to read 2x b/c when i read 'instance', i thought 'instance of object/class'. not super important. also not in love w word 'situation' as in the suggestion, if you have alternatives.

**Review by mongodben — COMMENTED** [review]
generally looks good. couple small things before approval. 

amazed by how minimal the changes needed are.

File: source/tutorial/swiftui.txt (hunk 6 above)
Comment (dacharyc) [reply source/tutorial/swiftui.txt:386]: That's fair - went with scenario.

**Review by mongodben — APPROVED** [review]
lgtm! nice work.

**Pull request merged**